ideascube/tests/test_models.py

```diff
@@ -72,10 +72,19 @@ def test_user_data_fields_should_return_labels_and_values(settings):
 class JSONModel(models.Model):
     class Meta:
         app_label = 'ideascube'
+        managed = False
 
     data = JSONField()
 
 
+@pytest.fixture()
+def json_table():
+    from django.db import connections
+    cursor = connections['default'].cursor()
+    cursor.execute("CREATE TABLE ideascube_jsonmodel "
+                   "(id, data)")
```

Comment:
So, I created this model just for the purpose of testing the `JSONField`. I don't actually care about this model.

Your solution is fine for now, but what I'd like to do eventually is just drop this model entirely, and just test the field on its own. We just need to figure out how. :smile: